src/components/apiDelegation/offered/ChooseOrgPage/ChooseOrgPage.module.css

```diff
@@ -11,16 +11,32 @@
   .topText{
     font-size: 20px;
   }
+
+  .delegableOrgsContainer{
+    margin-bottom: 2rem;
+  }
+
+  .chosenOrgsContainer {
+    margin-bottom: 2rem;
+  }
 }
 
 @media only screen and (min-width: 769px) {
-  .accordionScrollContainer {
+  .actionBarScrollContainer {
     height: 300px;
     overflow-y: auto;
   }
 
   .searchSection{
-    width: 45%;
+    width: 47.5%;
```

Comment:
47.5 😅

Reply:
That's the number that makes accordion and searchSection aligned

Reply:
Yeah, I just thought it looked funny hehe